presto-docs/src/main/sphinx/connector/hive.rst

```diff
@@ -25,6 +25,29 @@ data warehouse. Hive is a combination of three components:
 Presto only uses the first two components: the data and the metadata.
 It does not use HiveQL or any part of Hive's execution environment.
 
+Requirements
+------------
+
+The Hive connector requires a Hive metastore service (HMS). The following
```

Comment:
We should remove the versions here, since we support all versions of the metastore (the Thrift protocol is compatible).